src/N3Parser.js

```diff
@@ -27,14 +27,16 @@ export default class N3Parser {
       this._readPredicateOrNamedGraph = this._readPredicate;
     // Support triples in other graphs
     this._supportsQuads = !(isTurtle || isTriG || isNTriples || isN3);
+    // Whether the log:isImpliedBy predicate is supported
+    this._isImpliedBy = options.isImpliedBy;
     // Support nesting of triples
     this._supportsRDFStar = format === '' || /star|\*$/.test(format);
     // Disable relative IRIs in N-Triples or N-Quads mode
     if (isLineMode)
       this._resolveRelativeIRI = iri => { return null; };
     this._blankNodePrefix = typeof options.blankNodePrefix !== 'string' ? '' :
                               options.blankNodePrefix.replace(/^(?!_:)/, '_:');
-    this._lexer = options.lexer || new N3Lexer({ lineMode: isLineMode, n3: isN3 });
+    this._lexer = options.lexer || new N3Lexer({ lineMode: isLineMode, n3: isN3, isImpliedBy: this._isImpliedBy });
```

Comment:
Suggestion: make the lexer do this by default? I.e., change lexer behavior?

Reply:
To clarify, is the suggestion that the lexer *always* returns the `abbreviation` type and then the parser decides whether to produce `log:implies` or `log:isImpliedBy`.

If so, can be done, but will be slightly messier to implement than this option.

Reply:
Ah no, keep whatever option is less messy. Looks good in the lexer the way it is!